COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) 2003-2006 Eelco Dolstra, Google Inc.
```

Comment:
I don't understand this reasoning? In your view, what is this file for? What would have if Eelco quits nixos? Will he be removed from this file?